test/WebSites/TagHelpersWebSite/Views/Home/About.cshtml

```diff
@@ -15,5 +15,5 @@
                                     Version = new Version(1, 1),
                                     CopyrightYear = 1990,
                                     Approved = true,
-                                    TagsToShow = 30 }" />
+                                    TagsToShow = 30 }"></website-information>
```

Comment:
Why force it to not be self closing in the TagHelper since you're writing it as self closing?